.circleci/config.yml

```diff
@@ -0,0 +1,129 @@
+
+commands:
+  init:
+    description: "Install"
+    steps:
+      - kubernetes/install
+      - aws-eks/update-kubeconfig-with-authenticator:
+          authenticator-release-tag: v0.5.1
+          cluster-name: ${AWS_EKS_CLUSTER_NAME}
+          cluster-authentication-role-arn: ${AWS_EKS_ROLE_ARN}
+      - helm/install-helm-client:
+          version: v3.3.4
+      - run:
+          name: Install helmfile
+          command: |
+            curl -L --fail --retry 3 -o ./helmfile https://github.com/roboll/helmfile/releases/download/v0.130.0/helmfile_linux_amd64 
```

Comment:
Why don't you download file directly on "/usr/local/bin" running curl as sudo? alternatively, isn't there a way to indicate to circleci to run everything as root by default?   for instance:
```
docker:
  - image: circleci/node:9
    user: root
```

Reply:
Doesnt work to curl directly to '/usr/local/bin' :/